### ISSUE TITLE
oddity with indent rule in eslint 4

### ISSUE THREAD

**Issue by gurpreetatwal** [eslint/eslint#9776]
**Tell us about your environment**

* **ESLint Version:** 4.14.0
* **Node Version:** 8.9.3
* **npm Version:** 5.5.1

**What parser (default, Babel-ESLint, etc.) are you using?**
Default 

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  env: {
    node: true,
  },
  parserOptions: {
    ecmaVersion: 6,
  },
  rules: {
    indent: ['error', 2, {SwitchCase: 1}],
  },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const doesnt = function() {

  return library.method(`
    some multiline string
  `, {
    prop: true,
  });

};

const works = function() {

  return library.method({
    prop: 1,
  }, {
    prop: 2,
  });

};

const wanted = function() {

  return library.method(`
    some multiline string
  `, {
      prop: true,
    });

};

```

```bash
 npx eslint test.js
```

**What did you expect to happen?**
No errors

**What actually happened? Please include the actual, raw output from ESLint.**
```
/home/gurpreet/temp/test.js
  6:1  error  Expected indentation of 6 spaces but found 4  indent
  7:1  error  Expected indentation of 4 spaces but found 2  indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.
```

**Comment by not-an-aardvark** [comment]
Closing as a duplicate of https://github.com/eslint/eslint/issues/9061

**Comment by gurpreetatwal** [comment]
whoops, my apologies for the dupe
